addons/website_sale/views/templates.xml

```diff
@@ -623,7 +623,7 @@
                                 <div class="js_product js_main_product mb-3">
                                     <div>
                                         <t t-call="website_sale.product_price" />
-                                        <small class="ml-1 text-muted o_base_unit_price_wrapper d-none" groups="website_sale.group_show_uom_price">
+                                        <small class="ml-1 text-muted o_base_unit_price_wrapper" groups="website_sale.group_show_uom_price">
```

Comment:
Why this change?

Reply:
It's a class selector to prevent displaying the element.
```
.d-none {
  display: none !important;
}
```

Reply:
I know what d-none does thanks 🤣 

I don't think this change is needed. That class is removed/readded by the JS code when necessary